bugsnag-plugin-android-exitinfo/detekt-baseline.xml

```diff
@@ -4,6 +4,7 @@
   <CurrentIssues>
     <ID>MagicNumber:TraceParser.kt$TraceParser$16</ID>
     <ID>MagicNumber:TraceParser.kt$TraceParser$3</ID>
+    <ID>MaxLineLength:TombstoneParser.kt$TombstoneParser$val stackFrame = Stackframe(tombstoneTraceFrame.functionName, tombstoneTraceFrame.fileName, tombstoneTraceFrame.relPc, null)</ID>
```

Comment:
It would be nice to rather split this line up I think (rather than adding it to the baseline)